### PR TITLE
Documentation additions (sample_qc, plus miscellaneous)

### PR DESCRIPTION
**Pull request by johnc1231**
I added an annotations table for sample_qc, added two missing entries from the variant_qc table, and fixed various formatting issues and warnings that building docs was emitting.

### CODE REVIEW

**Comment by johnc1231** [comment]
fixes #1078

File: python/hail/dataset.py [hunk 1]
```diff
@@ -2147,21 +2147,21 @@ def lmmreg(self, kinship_vds, y, covariates=[], global_root="global.lmmreg", va_
...
-        | ``global.lmmreg.fit.logDeltaGrid`` | Array[Double]        | values of :math:`\\mathit{ln}(\delta)` used in the grid search                                                                                        |
+        | ``global.lmmreg.fit.logDeltaGrid`` | Array[Double]        | values of :math:`\\mathit{ln}(\delta)` used in the grid search                                                                                       |
         +------------------------------------+----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
-        | ``global.lmmreg.fit.logLkhdVals``  | Array[Double]        | (restricted) log likelihood of :math:`y` given :math:`X` and :math:`\\mathit{ln}(\delta)` at the (RE)ML fit of :math:`\\beta` and :math:`\sigma_g^2`   |
+        | ``global.lmmreg.fit.logLkhdVals``  | Array[Double]        | (restricted) log likelihood of :math:`y` given :math:`X` and :math:`\\mathit{ln}(\delta)` at the (RE)ML fit of :math:`\\beta` and :math:`\sigma_g^2` |
```
Comment (jbloom22) [inline python/hail/dataset.py]: changing spacing in this table caused the table to fail to be created and disappear from the docs (there are escaped characters so right alignment looks unaligned raw)

File: python/hail/dataset.py [hunk 2]
```diff
@@ -2946,7 +2946,57 @@ def same(self, other):
...
+        +---------------------------+--------+----------------------------------------------------+
+        | ``rHetHomVar``            | Double | Het/HomVar ratio across all variants               |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``rInsertionDeletion``    | Double | Insertion/Deletion ratio across all variants       |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``dpMean``                | Double | Depth mean across all variants                     |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``dpStDev``               | Double | Depth standard deviation across all variants       |
+        +---------------------------+--------+----------------------------------------------------+
+        | ``gqMean``                | Double | The average genotype quality across all variants   |
```
Comment (jbloom22) [inline python/hail/dataset.py]: "Genotype quality mean across all variants" would be more parallel to the standard deviation one

File: python/hail/dataset.py [hunk 3]
```diff
@@ -3243,6 +3293,10 @@ def variant_qc(self):
         +---------------------------+--------+----------------------------------------------------+
         | ``dpStDev``               | Double | Depth standard deviation across all samples        |
         +---------------------------+--------+----------------------------------------------------+
+        | ``gqMean``                | Double | The average genotype quality across all samples    |
+        +---------------------------+--------+----------------------------------------------------+
```
Comment (jbloom22) [inline python/hail/dataset.py]: "Genotype quality mean across all samples" would be more parallel to the standard deviation one

**Comment by jbloom22** [comment]
Looks great.  Make those minor changes, make sure the table renders, and merge.

**Pull request merged**